dataprofiler/profilers/profile_builder.py

```diff
@@ -486,8 +486,7 @@ def __init__(self, data, samples_per_update=None, min_true_samples=0,
         self._min_sample_size = 5000
 
         # assign data labeler
-        # TODO: when available, update to use "unstructured_options"
-        self.options.structured_options.unstructured_text = TextProfilerOptions()
+        self.options.unstructured_options.text = TextProfilerOptions()
```

Comment:
Old code was a placeholder, now that you're implementing this, we can get rid of this line :)